www-servers/nginx-unit/nginx-unit-1.3.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="NGINX Unit is a dynamic web and application server."
```

Comment:
`DESCRIPTION` should not contain the package name and a dot in the end

Reply:
Will fix.